src/base/bittorrent/session.cpp

```diff
@@ -346,6 +347,7 @@ Session::Session(QObject *parent)
     pack.set_int(libt::settings_pack::ssl_listen, 0);
     // To prevent ISPs from blocking seeding
     pack.set_bool(libt::settings_pack::lazy_bitfields, true);
+    pack.set_int(libt::settings_pack::alert_queue_size, 10000);
```

Comment:
Make sense. actually I did fix the relation to the torrent size, which means even size 100 would work now.
But having much torrents whould result into much alerts during online time.

But torrent amount is maybe not the right one. Maybe we should use max connections as multiplicator because I guess the most messages are generated on connection base because not all torrents are active.
Did you have a better suggestion then torrent amount? If not we could just use it but I don't like it very much because it's not related to torrent count.